app/controllers/application_controller.rb

```diff
@@ -1,8 +1,11 @@
 class ApplicationController < ActionController::Base
+  include Pundit
   protect_from_forgery with: :exception
   helper_method :current_person
 
   def current_person
     @current_person ||= Person.find(session[:person_id]) if session[:person_id]
   end
+
+  alias pundit_user current_person
```

Comment:
Yeahhhhh we could do that. Feel like it isn't terribly clear when to use current_user and when to use current_person in that case tho. This makes me lean further over to having our person classed renamed to user. I just think somewhere down the road we could end up using current_person and current_user interchangeably and that would kind of suck.

With pundit_user no one gonna be calling that by mistake methinks.